packages/core/src/components/CardListItem/CardListItem.tsx

```diff
@@ -45,14 +16,31 @@ export type CardListItemProps = {
 export default function CardListItem(props: CardListItemProps) {
   const { children, selected, onSelect, loading, disabled, noPadding = false, ...rest } = props;
 
-  const content = noPadding ? (
-    <StyledCardContentNoPadding>{children}</StyledCardContentNoPadding>
-  ) : (
-    <StyledCardContent>{children}</StyledCardContent>
+  const content = (
+    <CardContent sx={{ marginBottom: (theme) => (noPadding ? 0 : `${theme.spacing(2)} !important`) }}>
+      {children}
+    </CardContent>
   );
 
   return (
-    <StyledCard variant="outlined" selected={selected} disabled={disabled} {...rest}>
+    <Card
+      variant="outlined"
+      {...rest}
+      sx={{
+        width: '100%',
+        borderRadius: 1,
```

Comment:
In my testing, `1` doesn't set the correct border radius. Setting it as below works as expected.

```suggestion
        borderRadius: (theme) => `${theme.spacing(1)}`,
```